### ISSUE TITLE
order phishlets

### ISSUE THREAD

**Issue by Evi1Grey5** [Evi1Grey5/Evilginx-3-PHISHLET-LAB#3]
Here you can leave your wishes, suggestions, which phishlet you would like to order. 
This section will be exclusively for ordering [without flood] 
The creation is for informational purposes only, without deadlines.

**Comment by fabrikano** [comment]
office have adfs issue 
budi.santoso@kemendag.go.id
careers@epson.com.au
a.zverkov@lincolninternational.ru
al51578@alumnos.itam.mx
agpourna@uth.gr
cic.belux@bmw-i.lu
bandi.abc@polimi.it

  u can test those emails it redirect to main adfs site not the phish ones

**Comment by Jandre224** [comment]
Yes can you please fix the office365 phishlets to include adds? As the captured cookies aren't working and it doesn't capture the username and password

**Comment by Evi1Grey5** [comment]
> Yes can you please fix the office365 phishlets to include adds? As the captured cookies aren't working and it doesn't capture the username and password

I can recommend using Evilginx + EvilPuppet + Interceptors. This will improve your work with this tool.

**Comment by Jandre224** [comment]
How do I do that? Can you provide guidance?

**Comment by Evi1Grey5** [comment]
> How do I do that? Can you provide guidance?

There is no flood in this section and it is in the nature of ordering PHISHLETs. Create a separate section with your question.

**Comment by Jandre224** [comment]
I've done that thank you.. waiting for your response

**Comment by Evi1Grey5** [comment]
> Yes can you please fix the office365 phishlets to include adds? As the captured cookies aren't working and it doesn't capture the username and password

check the updated / added Microsoft2 folder in the repository

**Comment by Jandre224** [comment]
Thank you! I just saw it, and I tried the phishlet on my evilginx, yet it still fails to capture cookies and credentials, the only thing that was captured was when I entered the user/email, and on terminal it got the username from API hostname (last name) but even then it didn't capture the cookies or email password and save it to sessions.. would very much appreciate it, if you could look into this

**Comment by whithey79** [comment]
He’s a fucking scammer, he just deleted my issues of me telling him his phishlets doesn’t work.
